packages/insomnia-testing/tsconfig.json

```
@@ -0,0 +1,10 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "include": [
+    "webpack.config.js",
+    "jest.config.js",
+    ".bablerc.js",
```

```suggestion
    ".babelrc.js",
```

Altho I can't seem to find a `.babelrc.js` file in this package at all, so maybe this is not needed at all?
Should that exist given that we had a `.babelrc` previously, or is that no longer needed?

---

good call.  it was there at some stage, but since we are just using the typescript compiler now via ts-loader it isn't needed anymore.  we were _just barely_ using it before, anyway.

---

resolved in https://github.com/dimitropoulos/insomnia/pull/6/commits/da6d731cdf4a1335a6cb9a970e26ac1a9e9761d0